lib/array.js

```diff
@@ -38,6 +38,60 @@ const map = (
   }
 };
 
+const NS_PER_SEC = 1e7;
```

Comment:
This should probably be `1e9` according to its name.
```suggestion
const NS_PER_SEC = 1e9;
```